atom/browser/net/atom_network_delegate.cc

```diff
@@ -241,6 +243,12 @@ int AtomNetworkDelegate::OnBeforeSendHeaders(
     net::URLRequest* request,
     const net::CompletionCallback& callback,
     net::HttpRequestHeaders* headers) {
+  auto cmd_line = base::CommandLine::ForCurrentProcess();
+  auto client_id = cmd_line->GetSwitchValueASCII(
+      switches::kDevToolsEmulateNetworkConditionsClientId);
+  if (!client_id.empty())
+    headers->SetHeader(
+        switches::kDevToolsEmulateNetworkConditionsClientId, client_id);
```

Comment:
The api wasnt working recently because of https://github.com/electron/brightray/pull/203 , before there was a default network interceptor for empty client id, hence we were able to emulate network conditions without the extra header. The id is required to obtain the right interceptor now, another benefit is that when both api network emulation and devtools throttling are done, the former will take precedence now.